pom.xml

```
@@ -296,6 +324,11 @@
                                     <include>org.checkerframework:checker-qual</include>
                                     <include>org.checkerframework:checker-compat-qual</include>
                                     <include>org.codehaus.mojo:animal-sniffer-annotations</include>
+                                    <include>com.fasterxml.jackson.core:*</include>
+                                    <include>com.fasterxml.jackson.dataformat:*</include>
+                                    <include>com.fasterxml.jackson.module:*</include>
```

We can be specific here as there is only one artifact per group id:
```suggestion
                                    <include>com.fasterxml.jackson.dataformat:jackson-dataformat-yaml</include>
                                    <include>com.fasterxml.jackson.module:jackson-module-scala_${scala.binary.version}</include>
```

---

done